Data.json spatial bounding box ordering not understood

**Describe the bug**
Project Open Data spec for spatial coverage field says "coordinates are specified in decimal degrees and in the order of: minimum longitude, minimum latitude, maximum longitude, maximum latitude" https://project-open-data.cio.gov/v1.1/schema/#spatial 

This seems to match the ordering in the validator https://github.com/magda-io/magda/blob/33f669e86fd9b5466d0ef704085f23313998d6c2/magda-scala-common/src/main/scala/au/csiro/data61/magda/spatial/GeoJsonValidator.scala#L33 

**To Reproduce**
Steps to reproduce the behavior:

1. Harvest from https://data-melbournewater.opendata.arcgis.com/data.json

**Actual behavior**
```
[ERROR] [09/04/2019 05:32:10.462] [indexer-akka.actor.default-dispatcher-348639] [akka.actor.ActorSystemImpl(indexer)] Failed to parse spatial data for dataset ds-melbourne-water-a1dd7f467e3f6f7721a97b3d422479acab126be92b90c440ba9ee363aeaa3656: Invalid latitude (y) range: 144.8816. Should be within -90 to +90
```

**Additional context**
Example from that data.json:
```
spatial: "144.8816,-38.0069,145.9823,-37.4808",
```
Which does not get changed on https://data.gov.au/api/v0/registry/records/ds-melbourne-water-a1dd7f467e3f6f7721a97b3d422479acab126be92b90c440ba9ee363aeaa3656?dereference=true&aspect=dcat-dataset-strings&optionalAspect=dcat-distribution-strings&optionalAspect=dataset-distributions&optionalAspect=temporal-coverage&optionalAspect=dataset-publisher&optionalAspect=source&optionalAspect=source-link-status&optionalAspect=dataset-quality-rating

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -34,6 +34,7 @@ Indexer:
 -   Fixed indexer throws an error when affiliatedOrganisation field is created
 -   Fixed indexer incorrect parsing bounding box data in spatialCoverage aspect
 -   Added auth when crawling the registry
+-   Fixed Data.json spatial bounding box ordering not understood
 
 Cataloging:
 

--- a/magda-scala-common/src/main/scala/au/csiro/data61/magda/model/Misc.scala
+++ b/magda-scala-common/src/main/scala/au/csiro/data61/magda/model/Misc.scala
@@ -178,7 +178,7 @@ package misc {
         case emptyPolygonPattern() => None
         case csvPattern(_) =>
           val latLongs = string.split(",").map(str => CoordinateFormat.convertStringToBigDecimal(str))
-          fromBoundingBox(Seq(BoundingBox(latLongs(0), latLongs(1), latLongs(2), latLongs(3))))
+          fromBoundingBox(Seq(BoundingBox(latLongs(3), latLongs(2), latLongs(1), latLongs(0))))
         case polygonPattern(polygonCoords, _) =>
           val coords = polygonCoords.split(",")
             .map { stringCoords =>

